grafana-plugin/src/plugin/GrafanaPluginRootPage.tsx

```diff
@@ -31,20 +31,34 @@ import { PluginSetup } from 'plugin/PluginSetup/PluginSetup';
 import { rootStore } from 'state/rootStore';
 import { useStore } from 'state/useStore';
 import { isUserActionAllowed } from 'utils/authorization/authorization';
-import { DEFAULT_PAGE } from 'utils/consts';
+import { DEFAULT_PAGE, getOnCallApiUrl } from 'utils/consts';
 import 'assets/style/vars.css';
 import 'assets/style/global.css';
 import 'assets/style/utils.css';
+import { FaroHelper } from 'utils/faro';
+import { useOnMount } from 'utils/hooks';
 
 import { getQueryParams, isTopNavbar } from './GrafanaPluginRootPage.helpers';
 
 import grafanaGlobalStyle from '!raw-loader!assets/style/grafanaGlobalStyles.css';
 
 export const GrafanaPluginRootPage = (props: AppRootProps) => {
+  useOnMount(() => {
```

Comment:
Is there any difference between this and a `useEffect` with `[]` params?

Reply:
Nope, this super simple hook is just to avoid putting [] as dependency array every time you want effect to run only on mount. And it has explicit naming